src/main/java/com/pnudev/librarysystem/controller/CategoryController.java

```diff
@@ -27,33 +28,37 @@ public class CategoryController {
     private final CategoryService categoryService;
 
 
+    @Secured({"ADMIN", "CLIENT"})
```

Comment:
Better use `@PreAuthorize` as it allows you to work with SpEL

Reply:
Refactored